pyproject.toml

```diff
@@ -30,10 +30,9 @@ dependencies = [
     "PartSeg",
     "magicgui",
     "matplotlib",
-    "napari",
+    "napari[all]",
```

Comment:
I know this is supposedly "bad practice", but for researchers who just want something with just a one line installation what is the best approach?